src/assets/css/dashboard.css

```diff
@@ -345,7 +346,6 @@ div[data-role=controlgroup] a.ui-btn-active {
     right: 0;
     bottom: 0;
     font-weight: 400;
-    background: rgba(0, 0, 0, 0.7);
```

Comment:
Text is unreadable with a light image.
![too_bright](https://user-images.githubusercontent.com/56478732/111842028-edad3b80-890f-11eb-9984-146724693f6a.png)